### ISSUE TITLE
Should we expand the scope of this project beyond Google's Gemini?

### ISSUE THREAD

**Issue by faetalize** [faetalize/zodiac#31]
The features of this UI could be used to interact with other LLMs, such as GPT, and self hosted ones.
Is that a feature you'd like to see?

**Comment by ZacharyZcR** [comment]
Yes, we need this feature, maybe you should make this project belike a execute program on Windows\Linux\Mac, use electron.

**Comment by ZacharyZcR** [comment]
In addition, i18n is an option
